microsoft-edge/webview2/concepts/browser-features.md

```diff
@@ -101,7 +101,12 @@ The following Microsoft Edge and Google Chrome settings webpages aren't availabl
 *  `edge://supervised-user-internals`
 *  `edge://version`
 
+<!-- ====================================================================== -->
+## Google Authentication
+
+Google Authentication is disabled in embedded webviews, which includes WebView2, due to a security policy set by Google.  See [Upcoming security changes to Google's OAuth 2.0 authorization endpoint in embedded webviews](https://developers.googleblog.com/2021/06/upcoming-security-changes-to-googles-oauth-2.0-authorization-endpoint.html).
```

Comment:
Can we move the primary actor in this case up to the front of the sentence? Just to make it extra clear to folks that complaining to us won't help?

```suggestion
Google has disabled Google Authentication in embedded webviews, which includes WebView2, due to a security policy they have set.  See [Upcoming security changes to Google's OAuth 2.0 authorization endpoint in embedded webviews](https://developers.googleblog.com/2021/06/upcoming-security-changes-to-googles-oauth-2.0-authorization-endpoint.html).
```